tests/plugins/test_koji.py

```diff
@@ -83,6 +83,9 @@ def getRepo(self, repo):
             return None
         return GET_REPO_RESPONSE
 
+    def ssl_login(self, cert, ca, serverca, proxyuser=None):
+        return True
```

Comment:
Should this verify the `ca` parameter has the expected value? (The constructor could be told what the expected value is.)